terraso_backend/apps/project_management/graphql/projects.py

```diff
@@ -288,7 +298,7 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         project_id = kwargs["id"]
         project = cls.get_or_throw(Project, "id", project_id)
         cls.remove_null_fields(kwargs, ["privacy", "measurement_units"])
-        if not user.has_perm(Project.get_perm("change"), project):
+        if not check_project_permission(user, "update_requirements", Context(project=project)):
```

Comment:
question: what does `update_requirements` mean here?

Reply:
[Consult the product spec](https://airtable.com/appDU4llBX3AicyTX/tblo1pU8zkQzRmFsF/viwt0gf1rs4e7130i?blocks=hide): 

update project requirements (privacy, units, required inputs, pinned note, etc)
> Set privacy
> Set units of measurements
> Set name, description
> Soil depth intervals
> Data collection requirement